fix-point/.gitignore

```diff
@@ -0,0 +1,3 @@
+.stack-work/
```

Comment:
This `.gitignore` file is not needed. It's enough to have single `.gitignore` per repository.